src/Symfony/Component/Form/Tests/ChoiceList/Factory/PropertyAccessDecoratorTest.php

```diff
@@ -63,6 +63,23 @@ public function testCreateFromChoicesPropertyPathInstance()
         $this->assertSame(array('value'), $this->factory->createListFromChoices($choices, new PropertyPath('property')));
     }
 
+    /**
+     * @group legacy
+     */
+    public function testCreateFromChoicesPropertyPathWithCallableString()
+    {
+        $choices = array(array('property' => 'value'));
+
+        $this->decoratedFactory->expects($this->once())
+            ->method('createListFromChoices')
+            ->with($choices, 'end')
+            ->will($this->returnCallback(function ($choices, $callback) {
+                return array_map($callback, $choices);
```

Comment:
You can simplify this test by simply returning a string `'RESULT'` and asserting whether the return value matches that string. We don't actually need to call `array_map()`, that only complicates understanding the test.